triage/models/estimators.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.linear_model import LogisticRegression
+
+from triage.models.transformers import DsappCutOff
+
+class DsappLogisticRegression(BaseEstimator, ClassifierMixin):
```

Comment:
Interesting  idea, but we want allow the triage's user be able of using the simple `LogisticRegression` too. So, I would prefer your previous idea of moving that code to the `utils.features` module